xbmc/guilib/FFmpegImage.cpp

```diff
@@ -488,9 +537,9 @@ bool CFFmpegImage::DecodeFrame(AVFrame* frame, unsigned int width, unsigned int
   AVPixelFormat pixFormat = ConvertFormats(frame);
 
   SwsContext* context = sws_getContext(m_originalWidth, m_originalHeight, pixFormat, width, height,
-                                       AV_PIX_FMT_RGB32, SWS_BICUBIC, NULL, NULL, NULL);
+                                       dstFormat, SWS_BICUBIC, NULL, NULL, NULL);
 
-  if (range == AVCOL_RANGE_JPEG)
+  if (false && range == AVCOL_RANGE_JPEG)
```

Comment:
Is this intentionally in here?

Reply:
Ouch. I did some testing, but forgot to push the final version.

I guess my coding license should be revoked after 2 am :).